packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -34,11 +34,51 @@ storiesOf('DialogTrigger', module)
     'type: popover',
     () => render({type: 'popover'})
   )
+  .add(
+    'type: popover, small',
+    () => render({type: 'popover', size: 'S'})
+  )
+  .add(
+    'type: popover, medium',
+    () => render({type: 'popover', size: 'M'})
+  )
+  .add(
+    'type: popover, large',
+    () => render({type: 'popover', size: 'L'})
+  )
+  .add(
+    'type: popover, fullscreen',
+    () => render({type: 'popover', size: 'fullscreen'})
+  )
+  .add(
+    'type: popover, fullscreenTakeover',
+    () => render({type: 'popover', size: 'fullscreenTakeover'})
+  )
   .add(
     'type: modal',
     () => render({type: 'modal'}),
     {chromaticProvider: {scales: ['medium'], height: 1000}}
   )
+  .add(
+    'type: modal, small',
+    () => render({type: 'modal', size: 'S'})
+  )
+  .add(
+    'type: modal, medium',
+    () => render({type: 'modal', size: 'M'})
+  )
+  .add(
+    'type: modal, large',
+    () => render({type: 'modal', size: 'L'})
+  )
+  .add(
```

Comment:
this one is missing its rounded corners

Reply:
Ah, misunderstood the ticket, will bring back the rounded corners